modules/web/src/assets/config/changelog.json

```diff
@@ -1,193 +1,4 @@
 {
   "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.22.md",
```

Comment:
Should we also update version in this URL?

Reply:
There is not going to be a 2.23 changelog for quite some time. Do you think it's okay to have a 404 url?

Reply:
ah, if its going to land on 404 then its fine to keep it like this.